week-5-serving-fastapi/app.py

```diff
@@ -0,0 +1,73 @@
+import os
+from typing import List, Text, Tuple
+from pathlib import Path
+import torch
+from transformers import BertTokenizer
+from model import RegressionModel
+import gdown
+
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+
+MODEL_URL = "https://drive.google.com/file/d/1M5SFB5cYS7Q3oLpkEQVGERXtquwPxury/view?usp=sharing"
+MODEL_SAVE_PATH = "./weights/bert-base-uncased-regression-weights.pth"
+
+
+def init_model() -> Tuple[torch.nn.Module, torch.nn.Module, torch.device]:
```

Comment:
should we re-use this from [PR](https://github.com/makingbigdato/prjctr-02-mlop-infrastructure/pull/23)

Reply:
It is good idea to reuse the existing code, but my idea here was to atomize (separate and make independent) different tasks as much as possible, so to have a complete and independent examples for streamlit and fastapi.

Reply:
sounds good, just don't do this in real-life)